certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1785,6 +1785,25 @@ public class Script : ScriptBase
       acceptHeaderValue = "application/pdf";
     }
 
+    if ("GetDocumentsWithDocumentId".Equals(this.Context.OperationId, StringComparison.OrdinalIgnoreCase))
+    {
+      var uriBuilder = new UriBuilder(this.Context.Request.RequestUri);
+
+      acceptHeaderValue = "application/pdf";
+      string[] path = uriBuilder.Path.Split('/');
+      string documentId = HttpUtility.UrlDecode(path[8]);
```

Comment:
lets see if there is different way opposed to working with hardcoded array indices. looks fragile.